exporters/auto/src/main/java/com/google/cloud/opentelemetry/auto/GoogleCloudSpanExporterFactory.java

```diff
@@ -0,0 +1,25 @@
+package com.google.cloud.opentelemetry.auto;
+
+import com.google.cloud.opentelemetry.trace.TraceExporter;
+import io.opentelemetry.javaagent.tooling.TracerInstaller;
+import io.opentelemetry.javaagent.tooling.exporter.ExporterConfig;
+import io.opentelemetry.javaagent.tooling.exporter.SpanExporterFactory;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GoogleCloudSpanExporterFactory implements SpanExporterFactory {
+
+    private static final Logger log = LoggerFactory.getLogger(TracerInstaller.class);
+
+    @Override
+    public SpanExporter fromConfig(ExporterConfig _config) {
+        try {
+            return TraceExporter.createWithDefaultConfiguration();
+        } catch (IOException ex) {
+            log.error("Unable to create Google Trace exporter.", ex);
+            return null;
```

Comment:
There is no check for null in TracerInstaller, and it will throw an NPE in the line 168 AFAIU:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/266b8b927d559ac334962756ed8ecfd0effdda5d/javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/TracerInstaller.java#L161-L168

Might be better to just wrap exception into a RuntimeException and throw it here?